### ISSUE TITLE
I got DOTA 2 banned on my almost 5 years one and only steam account…

### ISSUE THREAD

**Issue by SCROLLYACHE32** [ValveSoftware/Dota2-Gameplay#6483]
### Description

Hello, I would like to share a story, but first, let's get to know my name is Artur, I'm 25 years old, I live in Ukraine, I've been playing DotA for about 2-3 years, at first I had 6k integrity ( I have a weak PC), but after Russia's attack on Ukraine, I had to very often leave from games because of sirens and in half a year my decency from 8k became 1k for a long time I can’t raise it because I often played mostly low priority (sd mode), it was uncomfortable for me to play with eternal mute, the players there, to put it mildly, are very toxic and not friendly and there is no communication due to mutes, then I got 1 decency .. I didn’t know how to raise it because my PC was very weak and I often didn’t load or crashed, here’s my story for whom it’s not difficult like the post so that support can see and unblock.. thanks for your attention steam link https://steamcommunity.com/id/ioiooqwedcaz/

I want to add that I'm not a booster I don't have any twink accounts I play for my own pleasure I have a crusader rank of 1

everything that was applied to me: Hello,

Thank you for submitting additional information about your Dota 2 ban. This information will be forwarded to the Dota 2 team.

Please note that you will not receive another response to this help request. Even though the Dota 2 team is unable to respond to individual inquiries, they review help requests submitted through this path.

Please feel free to respond to this help request with any other information you would like to share about your ban. Even though help request tickets are automatically closed after 14 days with no activity, the contents of your help request will remain available for review by the Dota 2 team and will remain associated with your account.

### Screenshots

![ASDDASDSAD213DASADS](https://user-images.githubusercontent.com/122224207/211209114-4a649ab2-7405-49df-a2df-603996d48b11.PNG) 
![dasdasdsadsadsaadsdas](https://user-images.githubusercontent.com/122224207/211209130-a7a159c5-7af2-407c-b2ce-2cd8ca2609fe.PNG)

![dasadsadsdasdsadsaadsdas123123](https://user-images.githubusercontent.com/122224207/211209141-282dc91a-baf3-41b1-b59f-f633d095bd9e.PNG)
![dasdsadsadsaddcsaadsdasdas123132](https://user-images.githubusercontent.com/122224207/211209143-823ccc5b-d8d7-44a9-ba83-5bacc8b21add.PNG)
